lib/razorpay/customer.rb

```diff
@@ -38,5 +38,21 @@ def fetchToken(tokenId)
     def deleteToken(tokenId)
       self.class.request.delete "#{id}/tokens/#{tokenId}" 
     end
+
+    def addBankAccount(options = {})
```

Comment:
general practice in ruby is to use snake case. In this sdk as well, we have mostly used snake case. Can we follow the same naming convention here as well ?

Reply:
updated functions to snake case and removed chaining